app/lint-baseline.xml

```diff
@@ -814,6 +814,17 @@
             column="17"/>
     </issue>
 
+    <issue
```

Comment:
Had to ignore this error as our min API version is 16 and this attribute was introduced in API 17.